package.json

```diff
@@ -115,6 +116,7 @@
     "fs-extra": "^9.0.1",
     "glob": "^7.1.6",
     "mocha": "^8.1.3",
+    "rimraf": "^3.0.2",
```

Comment:
what is this?

Reply:
Basically `rm -rf` in node js.
This way it works on windows as well.